tilequeue/rawr.py

```diff
@@ -446,3 +495,29 @@ def __call__(self, tile):
 
         body = response['Body']
         return unpack_rawr_zip_payload(self.table_sources, body)
+
+
+class RawrStoreSource(object):
+
+    """Rawr source to read from a tilequeue store."""
+
+    def __init__(self, store, table_sources, io_pool=None):
+        self.store = store
+        self.table_sources = table_sources
+        self.io_pool = io_pool
+
+    def _get_object(self, tile):
+        coord = unconvert_coord_object(tile)
+        format = zip_format
+        layer = 'rawr'
+        payload = self.store.read_tile(coord, format, layer)
+        return payload
+
+    def __call__(self, tile):
+        if self.io_pool:
+            payload = self.thread_pool.apply(
```

Comment:
Should this be `self.io_pool`? And ditto for above `RawrS3Source`?

And does this return the result itself? Or an async result where we have to call `.get()` on it? And since this is just a single call, is it worth sending a single job to a separate thread only to have to immediately wait for it? Why not just do it in the current thread itself? Apologies if I missed something here.

Reply:
Good point! Removed the `io_pool` stuff in f01b007c5166a66472a91076bedbaf2ac983ba9a.